app/Http/Controllers/UsersController.php

```diff
@@ -249,11 +358,67 @@ public function scores($_userId, $type)
         return response($json, is_null($json['error'] ?? null) ? 200 : 504);
     }
 
+    /**
+     * Get Own Data
+     *
+     * Similar to [Get User](#get-user) but with authenticated user (token owner) as user id.
+     *
+     * ---
+     *
+     * ### Response format
+     *
+     * See [Get User](#get-user).
+     *
+     * @authenticated
+     *
+     * @urlParam mode [GameMode](#gamemode). User default mode will be used if not specified. Example: osu
+     *
+     * @response "See User object section"
+     */
     public function me($mode = null)
     {
         return static::show(auth()->user()->user_id, $mode);
     }
 
+    /**
+     * Get User
+     *
+     * This endpoint returns the detail of specified user.
+     *
+     * ---
+     *
+     * ### Response format
+     *
+     * Returns [User](#user) object.
+     * Following attributes are included in the response object when applicable.
+     *
+     * Attribute                            | Notes
+     * -------------------------------------|------
+     * account_history                      | |
+     * active_tournament_banner             | |
+     * badges                               | |
+     * favourite_beatmapset_count           | |
+     * follower_count                       | |
+     * graveyard_beatmapset_count           | |
+     * groups                               | |
+     * loved_beatmapset_count               | |
+     * monthly_playcounts                   | |
+     * page                                 | |
+     * previous_usernames                   | |
+     * rankHistory                          | For specified mode.
+     * ranked_and_approved_beatmapset_count | |
+     * replays_watched_counts               | |
+     * scores_first_count                   | For specified mode.
+     * statistics                           | For specified mode. Inluces `rank` and `variants` attributes.
+     * support_level                        | |
+     * unranked_beatmapset_count            | |
+     * user_achievements                    | |
```

Comment:
I think we need a better way of documenting includes rather than just chucking them a ll in here, though